app/_includes/md/about-plugins.md

```diff
@@ -26,6 +26,44 @@ are fully compatible with each mode. See [Plugin Compatibility](/konnect/compati
 for a comparison.
 
 
+## Precedence 
+
+A plugin will always be run once and only once per request. But the
+configuration with which it will run depends on the entities it has been
+configured for.
+Plugins can be configured for various entities, combination of entities, or
+even globally. This is useful, for example, when you wish to configure a plugin
+a certain way for most requests, but make _authenticated requests_ behave
+slightly differently.
+
+Therefore, there exists an order of precedence for running a plugin when it has
+been applied to different entities with different configurations. The rule of
+thumb is: the more specific a plugin is with regards to how many entities it
+has been configured on, the higher its priority.
```

Comment:
```suggestion
been applied to different entities with different configurations. 
The general rule is: the more specific a plugin is with the number 
of entities it has been configured on, the higher its priority.
```
"rule of thumb" is too colloquial.

Also, I don't really understand what this sentence is trying to say. Is the plugin's priority higher when it has been configured on more entities? What exactly is "more specific" in this case?